README.md

```diff
@@ -42,3 +42,21 @@ To add a test, it would look like this.
 ```
 
 [GovDelivery Bulletins API](https://developer.govdelivery.com/api/comm_cloud_v1/Default.htm#API/Comm%20Cloud%20V1/API_CommCloudV1_Bulletins_CreateandSendBulletin.htm)
+
+## How to securely store GovDelivery username and password.
+
+Should be stored in local.settings.php, like so:
```

Comment:
```suggestion
Should be stored in settings.local.php, like so:
```

Reply:
I think that works for our implementation... but the more common implementation for others would likely be local.settings as that is usually the one to be used.

Reply:
Good catch